File: conversation.py
import openai
import os
from dotenv import load_dotenv

load_dotenv()
openai.api_key = os.getenv("API_KEY")

conversation = """
Human: {}
AI:
""".format(input("Human: "))

while True:
    response = openai.Completion.create(
      engine="text-davinci-001",
      prompt=conversation,
      temperature=0.8,
      max_tokens=60,
      top_p=1.0,
      frequency_penalty=0.5,
      presence_penalty=0.0,
      stop=["Human:"]
    )
    
    print("AI: ", end='')
    res = response.choices[0].text
    print(res)    
    conversation+=res
    conversation+="\nHuman: {}".format(input("Human: "))
    conversation+="\nAi: "

File: conversation.js
import promptSync from 'https://cdn.skypack.dev/prompt-sync'
import { Configuration, OpenAIApi } from 'https://cdn.skypack.dev/openai'
import { config } from 'https://deno.land/x/dotenv/mod.ts'
const env = config()

const prompt = promptSync()
const configuration = new Configuration({
  apiKey: env.API_KEY,
})
const openai = new OpenAIApi(configuration)

const input = prompt('Human: ')
let conversation = `
Human: ${input}
AI: 
`

while (true) {
  const response = await openai.createCompletion(
    'text-davinci-002',
    {
      prompt: conversation,
      temperature: 1.0,
      max_tokens: 50,
      top_p: 1.0,
      frequency_penalty: 0.0,
      presence_penalty: 0.0,
      stop: ['Human:']
    }
  )
  const res = response.data.choices[0].text
  console.log('AI: ' + res)
  conversation += res
  conversation += `\nHuman: ${prompt('Human: ')}`
  conversation += '\nAi: '
}